src/v2/Apps/FairOrginizer/Components/__tests__/FairOrganizerLatestArticles.jest.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import { graphql } from "react-relay"
+import { FairOrganizerApp_Test_Query } from "v2/__generated__/FairOrganizerApp_Test_Query.graphql"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { FairOrganizerLatestArticlesFragmentContainer } from "../FairOrganizerLatestArticles"
+
+jest.unmock("react-relay")
+
+const { getWrapper } = setupTestWrapper<FairOrganizerApp_Test_Query>({
+  Component: props => {
+    return <FairOrganizerLatestArticlesFragmentContainer {...(props as any)} />
+  },
+  query: graphql`
+    query FairOrganizerLatestArticles_Test_Query {
+      fairOrganizer(id: "example") {
+        ...FairOrganizerLatestArticles_fairOrganizer
+      }
+    }
+  `,
+})
+
+describe("FairOrganizerLatestArticles", () => {
+  it("renders correctly", () => {
```

Comment:
**suggestion** (non-blocking): I think that breaking this test into smaller tests will help clarify what "correctly" means for rendering this component. For example:

- it renders a section title with the fair name
- it renders a read all button
- it renders the most recent article in a large card
- it renders the next 6 articles in small cards

Reply:
Good point 👍  Thanks!